app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,70 @@
+class LikesController < ApplicationController
```

Comment:
Controllers all look great! Great job using scaffolding to create your rcav's :)